dspace-server-webapp/src/test/java/org/dspace/app/rest/matcher/CollectionMatcher.java

```diff
@@ -31,8 +31,7 @@ private CollectionMatcher() {
     public static Matcher<? super Object> matchCollectionEntry(String name, UUID uuid, String handle, Bitstream logo) {
         return allOf(
                 matchProperties(name, uuid, handle),
-                matchLinks(uuid),
-                matchLogo(logo)
+                matchLinks(uuid)
```

Comment:
please don't reduce the scope of the matcher as this would have the effect to reduce the usefulness of the existing tests. Instead, if a new match method that exclude embedded is required overload the method

Reply:
I agree with @abollini's feedback here. If we want a match method which doesn't check for a logo, let's create a separate `matchCollectionEntryNoLogo()` method (or similar).

Reply:
@abollini : This was resolved in a different manner.  A new `matchCollectionEntryFullProjection()` method was added, and all existing tests that depend on embeds (full projection) were updated to use that new method.